frontend/public/components/events.jsx

```diff
@@ -36,7 +36,14 @@ const kindFilter = (kind, {involvedObject}) => {
 
 class Inner extends React.PureComponent {
   render () {
-    const {klass, status, tooltipMsg, obj, lastTimestamp, message, source} = this.props;
+    const {klass, status, tooltipMsg, obj, lastTimestamp, firstTimestamp, message, source, count} = this.props;
+
+    let frequency;
```

Comment:
tiny nit: we've had a pattern of this style:

```JavaScript 
const frequency = count <= 1
  ? null 
  : <small className="co-sysevent__source">
    {count} times in the last <Timestamp timestamp={firstTimestamp} noTooltip={true} />
  </small>
```

Reply:
I believe this could even be

```jsx
const frequency = count > 1 && <small className="co-sysevent__source">
  {count} times in the last <Timestamp timestamp={firstTimestamp} noTooltip={true} />
</small>;
```

I mean even inline that below in the markup. That tends to be the style.

```jsx
{count > 1 && <small className="co-sysevent__source">
  {count} times in the last <Timestamp timestamp={firstTimestamp} noTooltip={true} />
</small>}
```